ast/Helpers.h

```diff
@@ -357,6 +357,14 @@ class MK {
         return Constant(loc, core::Symbols::T());
     }
 
+    static ExpressionPtr Bind(core::LocOffsets loc, ExpressionPtr value, ExpressionPtr type) {
+        return Send2(loc, T(loc), core::Names::bind(), loc, std::move(value), std::move(type));
+    }
+
+    static ExpressionPtr ClassOf(core::LocOffsets loc, ExpressionPtr value) {
+        return Send1(loc, T(loc), core::Names::classOf(), loc, std::move(value));
+    }
```

Comment:
What do you think about moving these helpers into `ClassNew.cc`? We can always extract them if we find they're useful elsewhere.

Reply:
(I'm happy to defer to trevor here, but fwiw we already have helpers for most other things like `T.let` and `T.cast` in this file. Seems fine to me to add these as well.)

Reply:
Yeah I think it's better to have all those helpers together here so it's easier to find them when needed and avoid duplication. WDYT?